doc/users/next_whats_new/simplify_font_setting_usetex.rst

```diff
@@ -0,0 +1,13 @@
+
+Simplifying the font setting for usetex mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now the :rc:`font.family` accepts some font names as value for a more
+user-friendly setup.
+
+.. plot::
```

Comment:
Since you don't plot anything, this doesn't show much; probably this should just be a code block.